pypeit/reduce.py

```diff
@@ -282,7 +287,7 @@ def extract(self, global_sky, sobjs_obj):
                 plate_scale = self.get_platescale(sobj)
                 # True  = Good, False = Bad for inmask
                 thismask = self.slitmask == sobj.SLITID  # pixels for this slit
-                inmask = (self.sciImg.fullmask == 0) & thismask
+                inmask = self.sciImg.boolean_mask(invert=True) & thismask
```

Comment:
Can we change the name of boolean_mask to boolean_gpm to consistent with our convention of trying to use either gpm or bpm everywhere.

Reply:
I've changed the name to `select_flag`, which I'm not crazy about either.  The point is that the method returns a boolean array that can be used to select pixels flagged with the specified bits.  I.e., it doesn't necessarily create a bpm or gpm, it creates the boolean mask you request.